### DISCUSSION TITLE
[suggestion] set piston speed

### DISCUSSION THREAD

**Discussion by TIKIRobo** [tsunamayo/Starship-EVO#4031] [Suggestions]
it would be nice if we could set piston speed, an example of a use for this would be a landing gear that has two pistons but only one actually pushing the foot we could set both pistons to be at say 5mps instead of having to fiddle around with % settings, this would be super useful for things like elevators that you don't want going too fast or whatever. It's not very important but instead a QOL thing for later down the line

**Comment by cloudrambler** [comment]
A related issue is that loaded pistons slow down, making it more difficult to match the speed of unloaded pistons. Having them slow down due to incumbent mass is fine, but maybe having an actual speed readout would be helpful, so we know what speed it is and then can use that value to match the other pistons.
